charts/premium/grafana/values.yaml

```diff
@@ -216,7 +216,7 @@ configmap:
             allowUiUpdates: false
             updateIntervalSeconds: 30
             options:
-              foldersFromFilesStructure: false
+              foldersFromFilesStructure: true
```

Comment:
To enable automatically creating Grafana folders based on subfolder. Teslamate has a lot of dashboards so i wanted to keep the main view neat.

Reply:
Did you check how this affects other charts?

Reply:
I would prefer not sneaking this into a teslamate PR tbh

Reply:
you would need to use `k8s-sidecar-target-directory` on the other charts for it to use subfolders otherwise everything just gets dumped into the root folder since the other charts end up in root.

We could update any other charts with more than a couple dashboards with `k8s-sidecar-target-directory` to put them in folders to keep them clean but i'll leave that up to whomever else feels it should happen. so far in the apps i use i havent found any that would need a folder.

i'll split it into its own PR

Reply:
moved to: https://github.com/truecharts/charts/pull/20461